src/main/java/com/exasol/dbbuilder/Table.java

```diff
@@ -14,7 +14,7 @@
     private final List<List<Object>> rows = new ArrayList<>();
 
     private Table(final Builder builder) {
-        super(builder.writer, builder.parentSchema, builder.name);
+        super(builder.writer, builder.parentSchema, builder.name, false);
```

Comment:
You use this constructor so many times with false. Can it be false by default and have two constructors? Or is it unclear?

Reply:
The simple reason is that I did not implement attaching to existing object for other types than Scripts yet.
At least attaching to tables should be implemented for the 1.0.0. Schemas maybe also. Everything else has lower priority in my eyes and can be added when needed.